packages/teams-js/webpack.config.cjs

```diff
@@ -73,14 +73,6 @@ module.exports = {
       PACKAGE_VERSION: JSON.stringify(packageVersion),
     }),
 
-    new DtsBundleWebpack({
```

Comment:
Does this mean the `dts` related package we consume in our root `package.json` can be removed?

Reply:
Yes, good catch. I forgot to fully remove it. Fixed!